jaeger-crossdock/src/main/java/com/uber/jaeger/crossdock/JerseyServer.java

```diff
@@ -49,6 +49,10 @@
 import org.glassfish.jersey.server.ResourceConfig;
 
 public class JerseyServer {
+  private static final String SAMPLING_HOST_PORT = "SAMPLING_HOST_PORT";
+  private static final String AGENT_HOST = "AGENT_HOST";
+  private static final String COLLECTOR_HOST_PORT = "AGENT_HOST";
```

Comment:
is this a typo or am I missing something?

Reply:
it's a typo, fixed. 

Currently it's not used, I wanted to make all this configurable.